src/uwu/getting-started.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Getting started
+lang: en-US
+sidebar: false
+---
+
+# Getting started
+
+## Installation
+
+::: tip Video tutorial
+ If you prefer using a video tutorial, Artesians made a beautiful one. You will find it... Oh oh, forgot we took it down 😭: we are really concerned by our users privacy.
+:::
+
+::: warning Warning
+  The installation is really difficult! Luckily we have a beautiful website which is fully PWA compliant. Forget about the app, you just need to press the "Install the app" notification on your phone. 
+:::
+
+### AltStore instructions for Mac
+1. Download **`Paperback.ipa`**
+1. Try to open it on your phone, that won't work.
+1. Just ask for help in #support
+
+## FAQ
+
+### How to install it on Android?
+Will the app work on Android? Maybe, ping Paper on Discord to check.
+
+### App crashes on launch
+It's a **bug**! Just go to #feedback channel on Discord and ask:
```

Comment:
```suggestion
It's a **feature**! Just go to #feedback channel on Discord and ask:
```

This one is totally optional. More of a programming joke more than anything. Feel free to ignore this one.
It's features, though! The crashes are intended!